monkey/infection_monkey/master/automated_master.py

```diff
@@ -26,16 +27,18 @@
 class AutomatedMaster(IMaster):
     def __init__(
         self,
+        island_address: str,
         current_depth: Optional[int],
         servers: Sequence[str],
         puppet: IPuppet,
-        control_channel: IControlChannel,
+        island_api_client: IIslandAPIClient,
```

Comment:
The whole point of having `IIslandAPIClient` is so that we don't leak any Island details into the Agent. `island_address` shouldn't be passed here.